Loop/Managers/Alerts/AlertManager.swift

```diff
@@ -202,8 +208,10 @@ public final class AlertManager {
                 repeats: false
             )
 
+            let intervalIdentifierValue = TimeInterval(minutes: minutes)
+
             let request = UNNotificationRequest(
-                identifier: "\(LoopNotificationCategory.loopNotRunning.rawValue)\(minutes)",
+                identifier: "\(LoopNotificationCategory.loopNotRunning.rawValue)\(intervalIdentifierValue)",
```

Comment:
I like minutes better too, but this is to avoid changing the existing identifiers in dev. Changing them means the old ones are not getting deleted, and people get spurious warnings when they update.